config/Shared/config_default.php

```diff
@@ -775,7 +775,6 @@
     'order-events' => MessageBrokerAwsConfig::SQS_TRANSPORT,
 ];
 
-$config[CartsRestApiConstants::IS_QUOTE_RELOAD_ENABLED] = true;
```

Comment:
you removed one line, but looks like there was a reason for this [https://github.com/spryker-shop/b2b-demo-shop/commit/1aff61b794305f9f7565ed63dff9f7c2f4b46681#diff-63fd0eb4016cfe314f73[…]53538807bf9b093df9f05a0eae](https://github.com/spryker-shop/b2b-demo-shop/commit/1aff61b794305f9f7565ed63dff9f7c2f4b46681#diff-63fd0eb4016cfe314f73fd68f8617c6bdb966353538807bf9b093df9f05a0eae)